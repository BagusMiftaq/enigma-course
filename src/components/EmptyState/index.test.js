import {render} from "@testing-library/react";

import EmptyState from "./index";

test("should render empty state component correctly", ()=>{
    const mockProps = {text: "List is empty"};
    const { asFragment } = render(<EmptyState {...mockProps}/>);

    expect(asFragment()).toMatchSnapshot();
})