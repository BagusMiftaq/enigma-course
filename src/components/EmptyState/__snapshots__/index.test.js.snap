// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render empty state component correctly 1`] = `
<DocumentFragment>
  <h4
    class="sc-bcXHqe hdXpYB"
  >
    List is empty
  </h4>
</DocumentFragment>
`;
